src/learn.jl

```diff
@@ -45,6 +45,24 @@ function log_value!(logger::LearnLogger, field::AbstractString, value)
     return value
 end
 
+"""
+    log_evaluation_row!(logger::LearnLogger, field::AbstractString, metrics::EvaluationRow)
+
+From fields in [`EvaluationRow`](@ref), generate and plot the composite [`evaluation_metrics_plot`](@ref)
+as well as `spearman_correlation` (if present).
```

Comment:
```suggestion
Generate, plot, and log the composite [`evaluation_metrics_plot`](@ref) of `metrics`, and log additional fields from [`EvaluationRow`](@ref) to be natively handled by `logger`.

For `TensorBoardLogger.TBLogger`, logs single additional `spearman_correlation` field (if not `missing`).
```
...not sure if this should be documenting what it currently does, vs what we'll want it to do when #55 eventually lands. maybe this vaguer approach is better? (can commit if so)

Reply:
I think we shouldn't single out `TensorBoardLogger.TBLogger`, since other loggers could implement `log_value!` if they wanted. I agree somewhat vague is good so that loggers can choose how to do it though. I would also drop the `::LearnLogger` type restriction, since that might make it easier to pass other loggers in.